src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,108 @@
+package duke;
+
+import duke.dukeexception.NoListException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.FileWriter;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents a wrapper class that deals with loading and saving tasks in a file.
+ */
+public class Storage {
+    private File file;
+
+    /**
+     * Constructs a Storage from a given filePath.
+     *
+     * @param filePath The given String filePath.
+     */
+    public Storage(String filePath) {
+        this.file = new File(filePath);
+        if (!this.file.getParentFile().exists()) {
+            this.file.getParentFile().mkdirs();
+        }
+    }
+
+    /**
+     * Converts a string that contains info about a Task into a Task.
+     *
+     * @param fullString The given string
+     * @return The Task with the appropriate information.
+     */
+    public static Task stringToTask(String fullString) {
+        String[] arr = fullString.split("\\|");
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = arr[i].trim();
+        }
+        Task curr;
+        if (arr[0].equals("T")) {
+            curr = new Todo(arr[2]);
+        } else if (arr[0].equals("E")) {
+            curr = new Event(arr[2], arr[3]);
+        } else {
+            curr = new Deadline(arr[2], arr[3]);
+        }
+        if (arr[1].equals("1")) {
+            curr.markAsDone();
+        }
+        return curr;
+    }
+
+    /**
+     * Converts a Task into a string.
+     *
+     * @param task THe given Task.
```

Comment:
Just a minor spelling mistake!
```suggestion
     * @param task The given Task.
```